sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -251,6 +257,13 @@ export { TokenCredential }
 
 export { TokenType }
 
+// @public
+export interface TracingOptions {
```

Comment:
I don't think I've ever noticed how funny this looks.

Also, now it makes me wonder why SpanOptions doesn't contain a field called spanOptions. :)

Reply:
Yeah, personally I think I prefer just putting the `tracingOptions?` field on whatever currently extends `TracingOptions`, since this only has one field and is now exposed to users when it probably wouldn't be used otherwise. But that's just my opinion :smile: